### ISSUE TITLE
Large system font prevents proper bluetooth dialog usage

### ISSUE THREAD

**Issue by molwizard** [corona-warn-app/cwa-app-android#560]
## Describe the bug
![corona_warn_app-font_size_issue](https://user-images.githubusercontent.com/30152414/84810648-eddc7b80-b00b-11ea-84d1-57a24661d527.jpg)

The bluetooth activation dialog that appears during the first application run if bluetooth is deactivated cannot be used properly if the system font size is very large (see attached screenshot). Not only there is a part of the dialog text invisible but also not all actions are visible . And the content of the dialog cannot be scrolled.
The app (version 1.0.0 from the Google Play Store) was installed on a Sony Xperia 5 with Android 9. It has only a medium system font size range. Other smartphones like the Samsung Galaxy S10e have a much wider system font size range. There even no action might be visible in the dialog. It is quite likely that this problem will affect older people (risk group members) because it is very likely that the system font size is increased on their smartphones.

## Expected behaviour

The complete dialog text is readable and all actions are accessible, independant of the system font size setting.

## Steps to reproduce the issue
1. Deactivate the bluetooth function.
2. Install the corona-warn-app.
3. Follow the initial setup instructions until the bluetooth activation dialog pops up. 

## Technical details
Sony Xperia 5 Smartphone
- Host Machine OS (Windows/Linux/Mac):
Android 9

## Possible Fix
Enable the scrollable property of the dialog window.

## Additional context

---
Internal Tracking ID: EXPOSUREAPP-1963

**Comment by tomjschwanke** [comment]
I believe this message is coming from Google Play Services too confirm enabling the API through an App

**Comment by DE-Max-42** [comment]
similar to #532 
Dupe?

**Comment by tomjschwanke** [comment]
> similar to #532
> duplicate?

That one is about the notification from GMS, this one here is a prompt (from I believe GMS too) about enabling the tracing functionality

**Comment by mynchau** [comment]
Thank you for reporting this bug and your patience awaiting a response. The team is getting it addressed so we can update you as soon as we know more. 
Best regards
MC 
Your Corona Warn-App Open Source Team

**Comment by dsarkar** [comment]
Hi @molwizard, @tomjschwanke, thanks again for contributing here. This issue seems fixed. We will close this issue now. Best regards, DS

---
Corona-Warn-App Open Source Team
